templates/alertmanagerconfig.go

```diff
@@ -36,26 +36,14 @@ func convertToAPIExtV1JSON(val interface{}) apiextensionsv1.JSON {
 var _false = false
 
 var pagerdutyAlerts = []string{
-	"NooBaaResourceErrorState",
-	"NooBaaNamespaceResourceErrorState",
-	"NooBaaBucketErrorState",
-	"NooBaaNamespaceBucketErrorState",
+	"DataSourceErrorState",
+	"BucketPolicyErrorState",
 }
 
 var smtpAlerts = []string{
-	"NooBaaBucketReachingQuotaState",
-	"NooBaaBucketExceedingQuotaState",
-	"NooBaaBucketLowCapacityState",
-	"NooBaaBucketNoCapacityState",
-	"NooBaaSystemCapacityWarning85",
-	"NooBaaSystemCapacityWarning95",
-	"NooBaaSystemCapacityWarning100",
+	"CacheBucketErrorState",
```

Comment:
can we add this alert to the pagerdutyAlerts list also as we discussed?

Reply:
We talked to Danny, and the conclusion from that meeting was the order that's there right now.

Reply:
I think Pranshu Naveen is correct, and Cache also needs to be part of pager duty alert. Also `"DataSourceErrorState",
"BucketPolicyErrorState"`
need to be part of SMTP alerts IMO.

Reply:
yes @cloudbehl , that's my point too..., because caching or default backing store is not part handled bt user directly and  namespacestore and bucketpolicy can create alerts if the user configures it wrong.